app/api/v1/utils/audio.py

```
@@ -0,0 +1,77 @@
+
+settings = get_settings()
+UPLOAD_DIR = settings.talk_audio_files_dir
+
+
+@router.post("/text-to-speech")
+async def text_to_speech(
+    request: Request,
+    agent_id: str = Form(...),
+    text: str = Form(...),
+    api_key: str = Depends(verify_token),
+    tts_service: TTSService = Depends(get_tts_service),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)